### DISCUSSION TITLE
#1000

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1000] [Discussion]

**Comment by Hans5958** [comment]
Moving this as a discussion and unlocked it, just so people can talk (even though IDK why it is locked). It's still the #1000.

:tada:

**Reply by Hans5958** [reply]
And unlocked again. Why can't we have nice things?

**Comment by retronbv** [comment]
YAYYAYAYAYAYAYYAYAAY #1000

**Comment by ghost** [comment]
🎉🎉🎈🥳🎉🎉🎉🎉🎉🎉🎈🥳🎉🎉🎈🥳🎉🎉

**Comment by ghost** [comment]
Should we also celebrate for #1024? It reads as 10000000000 in binary.

**Reply by pufferfish101007** [reply]
Absolutely! Especially as it's also 2^10, and also the number of bites in a megibyte.

**Comment by williamhorning** [comment]
#1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳 #1000!!!!!!!! 🥳

**Comment by Explosion-Scratch** [comment]
YAY! I remember being excited about #10!

**Reply by retronbv** [reply]
LOL

**Reply by GrahamSH-LLK** [reply]
Of course it's still open.

**Reply by ilbailba** [reply]
It was a success at the time, after all, wasn't it? :D

**Comment by Nonethousand** [comment]
wat

**Comment by ilbailba** [comment]
I think we should also celebrate the number of sa's founding year!
